src/main.cpp

```diff
@@ -66,10 +70,10 @@ void InitGyro()
     // Calculate an average offset for the gyro
     for (int i = 0; i < OFFSET_AVG_SAMPLES; i++)
     {
-    	gyroOffsetX += gyro.getGyroX();
-    	gyroOffsetY += gyro.getGyroY();
+        gyroOffsetX += gyro.getGyroX();
+        gyroOffsetY += gyro.getGyroY();
     }
-    
+    //use the average calculated while getting the samples to give a more accurate offset value
```

Comment:
Please follow spec with the rest of the comments. 

1. One space between "//" and the first letter of the sentence. 
2. Capitalize the first letter in the sentence.

Also, this doesn't make any sense.. "Use the average calculated..." The average calculated what? I suggest just writing something like, "Use the running average to calculate the offset."